client/src/pages/TournamentLobby.tsx

```diff
@@ -98,9 +98,15 @@ const TournamentLobby = () => {
 
   const startTournament = async () => {
     if (!inProgress && players.length > 1) {
-      setTournamentStarted(
-        await postStartTournament(socket.userID, tournamentCode),
+      const userConfirmed = window.confirm(
```

Comment:
I don't like this, if we were to add a confirmation we should use a custom modal. I think remove this for now.